src/styles/block/card.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  width: $card-width;
+
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  transition-duration: $transition-speed;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
+
+  &__name {
+    font-family: inherit;
+    display: flex;
+
+    font-size: 12px;
+    font-weight: 500;
+
+    line-height: 18px;
+    color: $color-text;
+
+    margin: 0;
+    padding-top: 40px;
+
+    transition-duration: $transition-speed;
+  }
+
+  &:hover &__name {
+    color: $color-title-hover;
+  }
+
+  &__code {
+    font-family: inherit;
+    font-size: 10px;
+    font-weight: 400;
+
+    margin: 0;
+    padding-top: 4px;
+
+    line-height: 14px;
+    color: $color-text-small;
+  }
+
+  &__reviews {
+    font-family: inherit;
+
+    display: flex;
+    justify-content: space-between;
+
+    padding-top: 18px;
+    padding-left: 2px;
+
+    &-text {
+      font-family: inherit;
+      font-size: 10px;
+      font-weight: 400;
+
+      margin: 0;
+      line-height: 14px;
+      color: $color-text;
+    }
+  }
+
+  &__price {
+    font-family: inherit;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 22px;
+
+    &-label {
+      font-family: inherit;
+      font-weight: 400;
+      font-size: 12px;
+
+      line-height: 18px;
+      color: $color-text-small;
+      margin: 0;
+    }
+
+    &-value {
+      font-family: inherit;
+      font-weight: 700;
+      font-size: 16px;
+
+      line-height: 18px;
+      color: $color-text;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    font-family: inherit;
+    font-size: 14px;
+    font-weight: 700;
+
+    padding: 0;
+    margin-top: 16px;
+
+    height: 40px;
+    width: 100%;
+
+    background-color: $color-button;
+    border-radius: 5px;
+    border: none;
+    transition-duration: $transition-speed;
+
+    color: $color-button-second;
+    line-height: 16px;
+
+    text-transform: uppercase;
+    text-align: center;
+
+    &:hover {
+      cursor: pointer;
+      border: 1px solid $color-button;
+      background-color: $color-button-second;
+      color: $color-button;
```

Comment:
The hover state changes the border and background color, which could affect the box model dimensions due to the 1px border. Make sure the box-sizing is set to border-box to include the border in the width and height calculations, or adjust the border-width to prevent layout shifts.